### ISSUE TITLE
supporting device - ILCE-7SM3

### ISSUE THREAD

**Issue by martinjhchoi** [dreambold/sony-language-tweak#1]
Hello, @dreambold 

Thank you for helping ppl via providing unlock codes.

I am using japanese domestic ver. of ILCE-7SM3 now.

(https://www.amazon.co.jp/-/en/ILCE-7SM3-Digital-Camera-a7SM3-without/dp/B08DV8RWRY?th=1)

Is there any way to unlock this camera and change language to English with your codes ? 

Hope you doing well, Thank you.

-Martin.

**Comment by dreambold** [comment]
Do you have an Ubuntu or mac machine?
If so, I can try to help you.

**Comment by martinjhchoi** [comment]
@dreambold Thank you for your response. 

first of all, I am not a SW developer. I just found you online while I am researching about unlocking Sony cameras. (you mentioned Ubuntu so I am worried so I am telling you that I am useless lol)

I am personally using MBP. that's it. 
Can it be possible ? :)

**Comment by dreambold** [comment]
Yes, of course. Can you install Teamviewer and let me know? I can connect to your Macbook and help you with the tweaks.

**Comment by martinjhchoi** [comment]
@dreambold I emailed you ! Thanks!

**Comment by martinjhchoi** [comment]
did you get my email @dreambold  ? I am just checking. Thanks.

**Comment by dreambold** [comment]
I'm tied with something urgent at the moment, will be back to you soon!

**Comment by martinjhchoi** [comment]
@dreambold I see. Thank you for your update. you can email me or notify me anytime. Please stay healthy ! Thanks.
